### ISSUE TITLE
ImportError: No module named aux.observer

### ISSUE THREAD

**Issue by holdenlee** [facebookresearch/CommAI-env#28]
When trying to run with Python 2.7.12 from Anaconda on Windows 10, I get
```
Traceback (most recent call last):
  File "run.py", line 18, in <module>
    from core.environment import Environment
  File "C:\Users\oldhe\Dropbox\CS\ML\CommAI-env\src\core\environment.py", line 13, in <module>
    from core.task import StateChanged, MessageReceived, \
  File "C:\Users\oldhe\Dropbox\CS\ML\CommAI-env\src\core\task.py", line 12, in <module>
    from core.aux.observer import Observable
ImportError: No module named aux.observer
```

**Comment by ajabri** [comment]
Which command are you running and from which dir?

**Comment by ajabri** [comment]
Does this solve your problem?
http://stackoverflow.com/questions/338768/python-importerror-no-module-named

**Comment by holdenlee** [comment]
No. It appears that Windows doesn't recognize any folder named `aux`: https://social.technet.microsoft.com/Forums/windows/en-US/e22c021d-d188-4ff2-a4dd-b5d58d979c58/the-specified-device-name-is-invalid?forum=w7itprogeneral

**Comment by ajabri** [comment]
Fixed in e516bde
